tests/importer/test_importer.py

```diff
@@ -312,7 +312,7 @@ def test_import_file_throwing_exception(self):
         exception = Exception('Error thrown for Unit Test')
         exception_json = ImporterError(str(exception)).getJSON()
         importer = XlsImporter(ingest_api=self.mock_ingest_api)
-        importer._generate_spreadsheet_json = MagicMock(side_effect=exception)
+        importer.generate_json = MagicMock(side_effect=exception)
```

Comment:
If we're testing the importer, we really shouldn't be mocking any part of it.

Reply:
it's testing the throwing of error in import_file. =